components/var_dumper/advanced.rst

```diff
@@ -181,6 +181,94 @@ method. They also typically implement the
 them from re-implementing the logic required to walk through a
 :class:`Symfony\\Component\\VarDumper\\Cloner\\Data` object's internal structure.
 
+The :class:`Symfony\\Component\\VarDumper\\Dumper\\HtmlDumper` limits string
+length and nesting depth of the output. These options can be overriden by
+providing a third parameter when calling 
+:method:`dump(Data $data) <Symfony\\Component\\VarDumper\\Dumper\\DataDumperInterface::dump>`
+::
+
+    use Symfony\Component\VarDumper\Dumper\HtmlDumper;
+
+    $output = fopen('php://memory', 'r+b');
+
+    $dumper = new HtmlDumper();
+    $dumper->dump($var, $output, array(
+        'maxDepth' => 1,
+        'maxStringLength' => 160
+    ));
+
+    // Limit nesting to 1 level and string length to 160 characters (default)
+
+The output format of a dumper can be fine tuned by the two flags 
+``DUMP_STRING_LENGTH`` and ``DUMP_LIGHT_ARRAY`` which are passed as a bitmap
+in the third constructor argument. They can also be set via environment 
+variables when using
+:method:`assertDumpEquals($dump, $data, $message) <Symfony\\Component\\VarDumper\\Test\\VarDumperTestTrait::assertDumpEquals>`
+during unit testing. The flags can be configured in ``phpunit.xml.dist``.
+
+* If ``DUMP_STRING_LENGTH`` is set, then the length of a string is displayed
+  next to its content.
+
+::
```

Comment:
If you do not want to terminate the last sentence preceding a code example with a colon, you have to be more explicit and have to use `.. code-block:: php` instead of the double colon.